tests/test_token_regression.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+import os
+from os.path import exists
+
+from subscriber import podaac_access as pa
+import shutil
+from pathlib import Path
+
+@pytest.mark.regression
+def setup_module(module):
+    print('*****SETUP*****')
+    tokens = pa.list_tokens(pa.token_url)
+    for x in tokens:
+        pa.delete_token(pa.token_url, x)
+
+@pytest.mark.regression
+def teardown_module(module):
+    print('*****TEARDOWN*****')
+    tokens = pa.list_tokens(pa.token_url)
+    for x in tokens:
+        pa.delete_token(pa.token_url, x)
+
+# REGRESSION TEST CURRENTLY REQUIRES A .NETRC file for CMR/Data Download
+# token API can be found here: https://wiki.earthdata.nasa.gov/display/EL/API+Documentation
+# explore https://urs.earthdata.nasa.gov/documentation/for_integrators/api_documentation#/oauth/token
+@pytest.mark.regression
+def test_list_tokens():
+    tokens = pa.list_tokens(pa.token_url)
+    for x in tokens:
+        pa.delete_token(pa.token_url, x)
+
+@pytest.mark.regression
+def test_edl_getToken():
+    token = pa.get_token(pa.token_url)
+    assert token != ""
+    token = pa.refresh_token(token)
+    assert token != ""
+    tokens = pa.list_tokens(pa.token_url)
+
+    assert len(tokens) == 1
+    for x in tokens:
+        assert x != ""
+
+    assert True == pa.delete_token(pa.token_url, token)
+
+@pytest.mark.regression
+def test_edl_max_token():
```

Comment:
In order to really test the max token logic I think you need to call `create_token` directly three times

Reply:
yeah, this was an older use case- thanks for catching this. We don't "create" a token directly, we "get" a token. if no tokens exist, then we create one. i'm not sure this needs to be tested at this point, since we only create if a token doesn't exist. This would be more testing EDL functionality, not functionality we expect to see.